css/css-syntax-3/tokenizing-and-parsing/input-byte-stream/support/bom-charset15.css.headers

```diff
@@ -0,0 +1 @@
+Content-Type: text/html
```

Comment:
Why do most of these CSS files have a Content-Type: text/html? Per https://drafts.csswg.org/css-cascade/#content-type, this should cause them to be ignored (unless they're linked from a same-origin quirks-mode document, in which case we ignore the declared content-type). This looks like a persistent bug across a bunch of tests.